versions.json

```diff
@@ -234,6 +241,13 @@
             } 
         }
     },
+    "rocm": {
+        "ubuntu22.04": {
+            "version": "6.2.60204-1",
+            "url": "https://repo.radeon.com/amdgpu-install/6.2.4/ubuntu/jammy/amdgpu-install_6.2.60204-1_all.deb",
+            "sha256": "38ba7dbf74b2145e406d800772f7918b8cd53b3efb94f5f17cdea21497ef4dab"
+        }
+    },
```

Comment:
Thanks for adding the ROCm version. Could you please update the verstion to 6.2.4?

Also, the RCCL version built against ROCm 6.2.4 is 2.20.5 (https://github.com/ROCm/rccl/releases/tag/rocm-6.2.4). Could you please add one entry for RCCL as well?

Reply:
sure, I will add it. The naming is a bit confusing since the URL has reference to rocm 6.2.4 and downloaded file is called rccl-rocm-6.2.4.tar.gz. I will comment to clarify this is indeed rccl not rocm